Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -683,7 +760,7 @@ class SpringAnimation extends Animation {
   }
 }
 
-type ValueListenerCallback = (state: {value: number}) => void;
+type AnimatedListenerCallback = (state: Object) => void;
```

Comment:
I see, does keeping `(state: {value: number}) => void` in AnimatedWithChildren and using a different type for AnimatedValueXY works? If not I don't really have a better idea :(